### ISSUE TITLE
setupVars.conf duplicate values

### ISSUE THREAD

**Issue by edgd1er** [pi-hole/docker-pi-hole#691]
Pi-hole v5.1.2 Web Interface v5.1.1 FTL v5.2

### Platform

- OS and version:  debian buster

- Platform:  rpi 4/rpi 3/ubuntu 20.04 with latest docker engine

### Expected behavior

_A clear and concise description of what you expected to happen._
No duplicated lines in setupVars.conf 

### Actual behavior / bug

_A clear and concise description of what the bug is._
At each container restart, if conditional_forwarding env variables are set, they are added in setupVars.conf, not replaced.... 

### Steps to reproduce

1.  remove existing container: `docker-compose down -v`
2.  have a setupVars.conf with REV_SERVER_XX defined
```      1 REV_SERVER_CIDR=192.168.0.0/24
      1 REV_SERVER_DOMAIN=example.com
      1 REV_SERVER_TARGET=192.168.0.1
      1 REV_SERVER=true 
```

3. run container and wait for a complete start: `docker-compose up -d && docker-compose logs -f`
4. after completed start, run : `sort etc-pihole/setupVars.conf | uniq -c`
```
      .......
      1 PIHOLE_INTERFACE=eth0
      1 QUERY_LOGGING=true
      2 REV_SERVER_CIDR=192.168.0.0/24
      2 REV_SERVER_DOMAIN=example.com
      2 REV_SERVER_TARGET=192.168.0.1
      2 REV_SERVER=true
```

## Debug Token

- URL:  https://tricorder.pi-hole.net/sccjc2sboo

### Screenshots

_If applicable, add screenshots to help explain your problem._

### Additional context

_Add any other context about the problem here._

Bug is detected in docker but file to correct is in this project. 

When using a container ( but should also be reproduced with a plain install) and settings env variables as here under:                                                                
      - CONDITIONAL_FORWARDING=true
      - CONDITIONAL_FORWARDING_IP=192.168.0.1
      - CONDITIONAL_FORWARDING_DOMAIN=example.com
      - CONDITIONAL_FORWARDING_REVERSE=0.168.192.in-addr.arpa

at each container restart, REV_SERVER_* lines are added to setupVars.conf. After a while, here is a count of duplicates lines

     89 REV_SERVER_CIDR=192.168.0.0/24
     89 REV_SERVER_DOMAIN=example.com
     89 REV_SERVER_TARGET=192.168.0.1
     89 REV_SERVER=true

I suspect the problem is here: delete_settings is ran with env variables not with used variables
https://github.com/pi-hole/pi-hole/blob/6b536b7428a1f57ff34ddc444ded6d3a62b00a38/advanced/Scripts/webpage.sh#L228

to be more explicit
`delete_settings CONDITIONAL_FORWARDING` is called, when `delete_settings REV_SERVER` should be called.

**Comment by DL6ER** [comment]
I confirm. I wasn't aware of the Docker setup doing this. Converting the docker script from `CONDTIONAL_FORWARDING_` to `REV_SERVER_` will solve the issue,

**Comment by edgd1er** [comment]
@DL6ER, according to your last comment, and as I understood, the documentation should be changed to notify users of variables names changes. is that correct ?

For a smooth transition, this code existing in webapage.sh, function processDNSSetting L218 could be called or added oncontainer's start  ?

```
   if [[ "${CONDITIONAL_FORWARDING}" == true ]]; then
        # Convert legacy "conditional forwarding" to rev-server configuration
        REV_SERVER=true
        add_setting "REV_SERVER" "true"

        REV_SERVER_DOMAIN="${CONDITIONAL_FORWARDING_DOMAIN}"
        add_setting "REV_SERVER_DOMAIN" "${REV_SERVER_DOMAIN}"

        REV_SERVER_TARGET="${CONDITIONAL_FORWARDING_IP}"
        add_setting "REV_SERVER_TARGET" "${REV_SERVER_TARGET}"

        REV_SERVER_CIDR="${CONDITIONAL_FORWARDING_REVERSE}"
        if [ -z "${REV_SERVER_CIDR}" ]; then
            # Convert existing input to /24 subnet (preserves legacy behavior)
            # This sed converts "192.168.1.2" to "192.168.1.0/24"
            # shellcheck disable=2001
            REV_SERVER_CIDR="$(sed "s+\\.[0-9]*$+\\.0/24+" <<< "${REV_SERVER_TARGET}")"
        fi
        add_setting "REV_SERVER_CIDR" "${REV_SERVER_CIDR}"

        # Remove obsolete settings from setupVars.conf
        delete_setting "CONDITIONAL_FORWARDING"
        delete_setting "CONDITIONAL_FORWARDING_REVERSE"
        delete_setting "CONDITIONAL_FORWARDING_DOMAIN"
        delete_setting "CONDITIONAL_FORWARDING_IP"
    fi 
```

**Comment by PromoFaux** [comment]
This should now be fixed in 5.2.1 (it was fixed in the core repo, https://github.com/pi-hole/pi-hole/pull/3882)
